src/StdChains.sol

```diff
@@ -0,0 +1,66 @@
+// SPDX-License-Identifier: MIT
+pragma solidity >=0.6.2 <0.9.0;
+
+pragma experimental ABIEncoderV2;
+
+import "./Vm.sol";
+
+abstract contract StdChains {
+    VmSafe private constant vm = VmSafe(address(uint160(uint256(keccak256("hevm cheat code")))));
+
+    struct Chain {
+        // The chain name, using underscores as the separator to match `foundry.toml` conventions.
+        string name;
```

Comment:
This comment is misleading, since this `name` is actual the real name like "Optimism Goerli" but the key name used is `optimism_goerli`. Similarly, the `setChain` method needs another param to differentiate the mapping key name from the chain's name